### PR TITLE
MGMT-14603: Assisted-Installer: Add MCE operator CI jobs

### PR DESCRIPTION
**Pull request by eliorerz**
Will be tested as part of https://github.com/openshift/assisted-service/pull/5203

/cc @gamli75 @osherdp @adriengentil

### CODE REVIEW

File: ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml [hunk 1]
```diff
@@ -281,6 +281,38 @@ tests:
...
+      PACKET_PLAN: m3.large.x86
+    workflow: assisted-baremetal
+- as: e2e-metal-assisted-mce-4-10
+  optional: true
+  steps:
+    cluster_profile: packet-assisted
+    env:
+      ASSISTED_CONFIG: |
+        OLM_OPERATORS=mce
+        OPENSHIFT_VERSION=4.10
```
Comment (gamli75) [inline ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml:301]: why the openshift version =  4.10?

Comment (eliorerz) [reply ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml:301]: This is minimum version for MCE operator, I want to check it, but I wanted to add to assisted service , not test-infra..

**Review by adriengentil — APPROVED** [review]

**Pull request merged**